client/components/banner/test/index.jsx

```diff
@@ -1,165 +1,197 @@
+		expect( container.getElementsByClassName( 'plan-price__integer' ) ).toHaveLength( 0 );
 	} );
 
 	test( 'should render a .banner__description when description is specified', () => {
-		const comp = shallow( <Banner { ...props } description="test" /> );
-		expect( comp.find( '.banner__description' ) ).toHaveLength( 1 );
+		render( <Banner { ...props } description="test" /> );
+		expect( screen.getByText( 'test' ) ).toBeVisible();
 	} );
 
 	test( 'should not render a .banner__description when description is not specified', () => {
-		const comp = shallow( <Banner { ...props } /> );
-		expect( comp.find( '.banner__description' ) ).toHaveLength( 0 );
+		const { container } = render( <Banner { ...props } /> );
+		expect( container.getElementsByClassName( 'banner__description"' ) ).toHaveLength( 0 );
 	} );
 
 	test( 'should render a .banner__list when list is specified', () => {
-		const comp = shallow( <Banner { ...props } list={ [ 'test1', 'test2' ] } /> );
-		expect( comp.find( '.banner__list' ) ).toHaveLength( 1 );
-		expect( comp.find( '.banner__list li' ) ).toHaveLength( 2 );
-		expect( comp.find( '.banner__list li' ).at( 0 ).text() ).toContain( 'test1' );
-		expect( comp.find( '.banner__list li' ).at( 1 ).text() ).toContain( 'test2' );
+		const { container } = render( <Banner { ...props } list={ [ 'test1', 'test2' ] } /> );
+		expect( container.querySelectorAll( '.banner__list' ) ).toHaveLength( 1 );
+		expect( container.querySelectorAll( '.banner__list li' ) ).toHaveLength( 2 );
+		expect( screen.getByText( 'test1' ) ).toBeVisible();
+		expect( screen.getByText( 'test2' ) ).toBeVisible();
 	} );
 
 	test( 'should not render a .banner__list when description is not specified', () => {
-		const comp = shallow( <Banner { ...props } /> );
-		expect( comp.find( '.banner__list' ) ).toHaveLength( 0 );
+		const { container } = render( <Banner { ...props } /> );
+		expect( container.getElementsByClassName( '.banner__list' ) ).toHaveLength( 0 );
 	} );
 
 	test( 'should record Tracks event when event is specified', () => {
-		const comp = shallow( <Banner { ...props } event="test" /> );
-		expect( comp.find( 'TrackComponentView' ) ).toHaveLength( 1 );
+		renderWithProvider( <Banner { ...props } event="test" /> );
+		expect( recordTracksEvent.mock.calls[ 0 ][ 1 ].cta_name ).toBe( 'test' );
 	} );
 
 	test( 'should not record Tracks event when event is not specified', () => {
-		const comp = shallow( <Banner { ...props } /> );
-		expect( comp.find( 'TrackComponentView' ) ).toHaveLength( 0 );
+		renderWithProvider( <Banner { ...props } /> );
+		expect( recordTracksEvent ).not.toHaveBeenCalled();
 	} );
 
 	test( 'should render Card with href if href prop is passed', () => {
-		const comp = shallow( <Banner { ...props } href={ '/' } /> );
-		expect( comp.find( Card ) ).toHaveLength( 1 );
-		expect( comp.find( Card ).props().href ).toBe( '/' );
+		const { container } = render( <Banner { ...props } href={ '/' } /> );
```

Comment:
Nit: we can simplify the string prop as above. I see this a few more times in the PR, so it might make sense to go through it and update them all if you agree.